site/_guides/cert-manager.md

```diff
@@ -487,11 +505,108 @@ $ curl https://httpbin.davecheney.com/get
 
 ![httpbin.davecheney.com screenshot][httpbin]
 
+## Making cert-manager work with HTTPProxy
+
+cert-manager currently does not have a way to interact with HTTPProxy objects in order to respond to the HTTP01 challenge correctly.
+(See [#950](https://github.com/projectcontour/contour/issues/950) and [#951](https://github.com/projectcontour/contour/issues/951) for details.)
+cert-manager does this by creating a new, temporary Ingress object that will direct requests from Let's Encrypt to temporary pods called 'solver pods'.
+These pods know how to respond to Let's Encrypt's challenge process for verifying you control the domain you're issuing certificates for.
+This means that cert-manager can't be *directly* used for generating certificates for HTTPProxy configuration.
+
+However, we can create a dummy Ingress object that will have cert-manager provision the certificate Secret, so that we can consume it in a HTTPProxy.
+This works because Contour expects the Secrets to look the same anyway.
+
+To do this, we need to create our HTTPProxy and Ingress objects.
+
+This example uses the hostname `httpbinproxy.davecheney.com`, remember to create that name before starting.
+
+Firstly, the HTTPProxy:
+
+```yaml
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  name: httpbinproxy
+spec:
+  virtualhost:
+    fqdn: httpbinproxy.davecheney.com
+    tls:
+      secretName: httpbin
+  routes:
+  - services:
+    - name: httpbin
+      port: 8080
+
+```
+
+This config will be marked as Invalid by Contour, since the TLS secret doesn't exist yet.
+Once that's done, create the dummy Ingress object:
+
+```yaml
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    ingress.kubernetes.io/force-ssl-redirect: "true"
+    kubernetes.io/ingress.allow-http: "false"
```

Comment:
I think ingress.allow-http defaults to false, we shouldn't need this annotation.